tests/conversion/downcast-converters.js

```diff
@@ -686,6 +686,26 @@ describe( 'downcast-converters', () => {
 			// No attribute set.
 			expect( viewToString( viewRoot ) ).to.equal( '<div><p>foobar</p></div>' );
 		} );
+
+		it( 'should not convert or consume if element creator returned null', () => {
+			const spy = sinon.spy();
+
+			dispatcher.on( 'attribute:class', changeAttribute( () => {
+				spy();
+
+				return null;
+			} ) );
+
+			const modelElement = new ModelElement( 'paragraph', { class: 'foo' }, new ModelText( 'foobar' ) );
+
+			model.change( writer => {
+				writer.insert( modelElement, modelRootStart );
+			} );
+
+			expect( viewToString( viewRoot ) ).to.equal( '<div><p class="foo">foobar</p></div>' );
+
+			expect( spy.called ).to.be.true;
```

Comment:
AFAIR it's better to use `sinon.assert.called( spy )` - there were some issues with `expect()` & sinon.